doc/source/whatsnew/v3.0.0.rst

```diff
@@ -372,7 +372,7 @@ Period
 
 Plotting
 ^^^^^^^^
--
+- Bug in :meth:`MPLPlot._plot` ignoring parameter ``style`` when using error bar parameters ``yerr`` or ``xerr`` (:issue:`7023`)
```

Comment:
Same comment as in original PR, "Can you describe this in a user-facing way"?
I.E. the method that the user uses is not `MPLPlot._plot` but rather `DataFrame.plot` or `Series.plot`